extensions/vscode/webviews/homeView/src/style.css

```diff
@@ -8,6 +8,10 @@ body {
   padding: 0;
 }
 
+::selection {
+  background-color: rgba(195, 183, 184, 0.15);
+}
```

Comment:
This preserves the `input` selection color, and matches the highlight color across VS Code. This doesn't appear to be a variable, instead it is hardcoded which is interesting. I did check, it is consistent across themes.